src/middlewares/routerTransformCompactedPayloadV1.ts

```
@@ -0,0 +1,27 @@
+        input: body.input.map((input) => ({
+          ...input,
+          connection:
+            body.connections[`${input.metadata.sourceId}:${input.metadata.destinationId}`],
+          destination: body.destinations[input.metadata.destinationId],
+        })),
+        destType: body.destType,
+      } as RouterTransformationRequest;
+    }
+    await next();
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)